src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj

```diff
@@ -8,10 +8,6 @@
     <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Please add `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` here and fix build issues if any. We plan to enforce this soon across all test projects

Reply:
Done, also did for non-tests.  Let me know if that was wrong.

Reply:
Don't enable this for non-tests yet, there are pieces of generated code that still generate some warnings and we need some fixes there

Reply:
It had no warnings for me.  Should I revert or just leave?

Reply:
Done.